data-structures-linked-lists/get-length.js

```diff
@@ -0,0 +1,11 @@
+/* exported getLength */
+
+function getLength(list) {
+  let counter = 1;
+  let copy = list;
```

Comment:
Careful, this isn't a copy, since `list` is an object. In any case, a copy isn't necessary to solve this.